src/engine-actions/get-max-channel.js

```diff
@@ -8,16 +8,42 @@ const { listChannels } = require('../lnd-actions')
  * @return {String} res.maxBalance - the max balance in all open channels.
  */
 async function getMaxChannel ({ outbound = true } = {}) {
-  const { channels = [] } = await listChannels({ client: this.client })
+  const [
+    { channels = [] } = {},
+    { pendingOpenChannels = [] } = {}
+  ] = await Promise.all([
+    // The response from listChannels consists of channels that may be active or inactive
+    listChannels({ client: this.client }),
+    listPendingChannels({ client: this.client })
+  ])
   const balance = outbound ? 'localBalance' : 'remoteBalance'
 
-  if (channels.length === 0) {
-    this.logger.debug('getMaxChannel: No channels exist')
+  if (channels.length === 0 && pendingOpenChannels.length === 0) {
+    this.logger.debug('getMaxChannel: No open or pending channels exist')
     return {}
   }
 
-  const maxBalance = channels.reduce((max, channel) => Big(channel[balance]).gt(max) ? Big(channel[balance]) : max, Big(channels[0][balance]))
-  return { maxBalance: maxBalance.toString() }
+  const maxBalance = channels.reduce((max, channel) => {
+    if (Big(channel[balance]).gt(max)) {
+      return Big(channel[balance])
+    } else {
+      return max
+    }
+  }, Big('0'))
+
+  const maxPendingBalance = pendingOpenChannels.reduce((max, pendingChannel) => {
+    if (Big(pendingChannel.channel[balance]).gt(max)) {
```

Comment:
Something we might want to cleanup is that IF we used a map on `pendingOpenChannels` so that the structure was the same as `channels`, we could break out this reduce function into its own method `findMaxChannel` (since this call is `getMaxChannel`) and use it in both places. It could potentially be easier to test